ui/jobs/jobs.html

```diff
@@ -12,7 +12,7 @@
   <script src="../../resources/timericon.js"></script>
   <script src="../../resources/widgetlist.js"></script>
   <script src="../../resources/lang.js"></script>
-  <script src="../../resources/lang-en.js"></script>
+  <script src="../../resources/lang/en.js"></script>
```

Comment:
Can you include lang/ko.js here as well?

Reply:
Yep.